README.md

```diff
@@ -38,11 +38,47 @@ action "PHPStan" {
 You might want to add extra plugins to phpstan to be more strict. To install them define
 `COMPOSER_PACKAGES` as environment variable in your workflow.
 
+```diff
+workflow "Qa workflow" {
+  on = "push"
+  resolves = [
+    "PHPStan",
+  ]
+}
+
+action "PHPStan" {
+  uses = "docker://oskarstark/phpstan-ga"
+  args = "analyse src"
+  secrets = ["GITHUB_TOKEN"]
+  env = {
+    COMPOSER_PACKAGES = "phpstan/phpstan-phpunit"
+  }
+}
+```
+
 ## Additional Alpine packages
 The docker image contains a minimal set of installed php packages. For some projects you
 might want to add more php extensions. Define `PHP_EXTENSIONS` as environment variable in
 your workflow to do so.
 
+```diff
+workflow "Qa workflow" {
+  on = "push"
+  resolves = [
+    "PHPStan",
+  ]
+}
+
+action "PHPStan" {
+  uses = "docker://oskarstark/phpstan-ga"
```

Comment:
IMO this is only working when referencing the Dockerfile and not the built image

Reply:
I think it is not right what you are saying here. If you check the linked pr in phpdoc you will see that the packages are installed in the container. This will be done each run. It does slow down the run,but it works perfectly.

Reply:
That’s what I mean, it is only working when building the image all the time (on every commit)